sdk/go/pulumi-language-go/main.go

```diff
@@ -32,12 +32,14 @@ import (
 	"github.com/blang/semver"
 	pbempty "github.com/golang/protobuf/ptypes/empty"
 	"github.com/opentracing/opentracing-go"
+	"github.com/rogpeppe/go-internal/modfile"
```

Comment:
Why are we using this package instead of https://pkg.go.dev/golang.org/x/mod@v0.10.0/modfile?

Reply:
My bad. When I wrote this originally, I couldn't find golang.org/x/tools in the sdk/go.mod so I assumed we didn't want that dependency—not remembering at the time that modfile was in golang.org/x/mod even though I realized that later.

This module is a drop-in replacement, though, so I've switched over to it: https://github.com/pulumi/pulumi/pull/12755